src/Compilers/Core/CodeAnalysisTest/Text/StringTextTest.cs

```diff
@@ -226,7 +225,7 @@ public void FromStream_CheckSum_BOM()
         {
             var bytes = new byte[] { 0xef, 0xbb, 0xbf, 0x61, 0x62, 0x63 };
 
-            var source = SourceText.From(new MemoryStream(bytes), Encoding.ASCII);
+            var source = SourceText.From(new MemoryStream(bytes), Encoding.UTF8);
```

Comment:
As someone who has previously broken this functionality I think we need to keep this ASCII.  That should be fine in CoreCLR though.  Not sure why it needed to change here.